src/index.js

```diff
@@ -356,11 +325,11 @@ export class FlashlsHandler {
 
     let buffered = this.tech_.buffered();
 
-    if (buffered.length === 1) {
-      removeCuesFromTrack(0, buffered.start(0), this.inbandTextTrack_);
-      removeCuesFromTrack(buffered.end(0), Infinity, this.inbandTextTrack_);
-    } else {
-      removeCuesFromTrack(0, Infinity, this.inbandTextTrack_);
+    if (!buffered.length) {
+      Object.keys(this.inbandTextTracks_).forEach((key) => {
+        removeCuesFromTrack(0, Infinity, this.inbandTextTracks_[key]);
```

Comment:
Do we want to still clear our inband text tracks in portions of the buffer that may no longer exist after seek (if part of the buffer is cleared out), or is that not an issue?

Reply:
Was that an issue? I can add it back, wasn't quite sure what that was for.

Reply:
I'm not sure myself, but the code seems intentional.

Reply:
I'll add it back, it makes sense, if the data gets removed from the buffer, then its going to have to be reprocessed anyway and will get added. I do remember that we had issues with too many cues for a live stream so we added `removeOldCues`, this was probably done for a similar reason